extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,107 @@
-  <a href="http://www.mozilla.org/about/forums/">newsgroup</a>.
+    <span class="heading">No abusing people</span>.
+    Constant and intense critique is one of the reasons we build great products. 
+    It's harder to fall into group-think if there is always a healthy amount of
+    dissent. We want to encourage vibrant debate inside of the Mozilla 
+    community, we want you to disagree with us, and we want you to effectively 
+    argue your case. However, we require that in the process, you criticize 
+    <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
+    algorithms, and schedules. Examples of people include: developers,
+    designers and users. <b>Attacking or encouraging attacks on a person 
+    may result in you being banned from [% terms.Bugzilla %]</b>.  
   </li>
 
   <li>
-  <span class="heading">No obligation</span>.
-  "Open Source" is not the same as "the developers must do my bidding."
-  Everyone here wants to help, but no one else has any <i>obligation</i> to fix
-  the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
-  expect someone to fix a [% terms.bug %] by a particular date or release.
-  Aggressive or repeated demands will not be received well and will almost
-  certainly diminish the impact and interest in your suggestions.
+    <span class="heading">No obligation</span>.
+    "Open Source" is not the same as "the developers must do my bidding."
+    Everyone here wants to help, but no one else has any <i>obligation</i> to fix
+    the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
+    expect someone to fix a [% terms.bug %] by a particular date or release.
+    Aggressive or repeated demands will not be received well and will almost
+    certainly diminish the impact and interest in your suggestions.
+  </li>
+  
+  <li>
+    <span class="heading">No spam</span>.
+    Posting comment spam will lead to the suspension of your account.
   </li>
 
   <li>
-  <span class="heading">No abusing people</span>.
-  Constant and intense critique is one of the reasons we build great products.
-  It's harder to fall into group-think if there is always a healthy amount of
-  dissent. We want to encourage vibrant debate inside of the Mozilla
-  community, we want you to disagree with us, and we want you to effectively
-  argue your case. However, we require that in the process, you attack
-  <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
-  algorithms, and schedules. Examples of people include: developers,
-  designers and users. <b>Attacking a person may result in you being banned
-  from [% terms.Bugzilla %].</b>
+    <span class="heading">No pointless comments</span>.
+    Limit comments on a [% terms.bug %] to information which will help with 
+    resolving it. Unless requested, additional "I see this too" or "It works for me" 
+    comments are unnecessary unless they are on a different platform or a significantly 
```

Comment:
I would omit "or a significantly different build", because having people add "still buggy" every new release (significantly different build!) is pointless, too. People who compile their own Firefoxen and see e.g. a graphics issue when building with a different library than the default will know whether that's useful or not and comment anyway (plus that could broadly be considered "platform").